content/blog/devmode-vs-prodmode.md

```diff
@@ -0,0 +1,31 @@
+
+# Production Mode
+Production mode is tailored for environments where stability, reliability, and scalability are paramount. The Orchestrator Helm chart or Orchestrator operator is designed specifically to meet the demanding requirements of production environments. Key requirements include:
+
+* **Long-running Workflows**: Production mode supports workflows that may take several hours or even days to complete, ensuring that these processes run smoothly without interruption.
+* **Persistence of Running Workflow Instances**: Ensuring the persistence of workflow instances is critical. In production, all workflow data is stored and maintained even if the orchestrator restarts, preventing data loss.
+* **Event Handling Reliability**: Reliable handling of events is essential for maintaining workflow integrity and ensuring that all triggers and actions occur as expected.
+* **Scalability**: The system must be capable of scaling up to handle increasing workloads, allowing for the addition of resources as demand grows.
+* **Updates Through Pipelines**: Workflows can be updated and managed through continuous integration/continuous deployment (CI/CD) pipelines, facilitating smooth and efficient updates.
+* **Externalizing Credentials**: Credentials are managed outside the workflow configuration to enhance security and simplify updates.
+* **Runtime Isolation**: Each workflow runs in its isolated environment, preventing any interference between workflows.
+* **Authorization and Administration**: Robust mechanisms are in place for authorizing and administering workflow deployments, ensuring that only authorized personnel can make changes.
+
+# Development (Dev) Mode
+Dev mode, as the name implies, is optimized for development purposes. It allows developers to experiment with the Orchestrator plugins without the need for a full deployment process or access to a Kubernetes (K8s) or OpenShift (OCP) cluster. Characteristics of dev mode include:
+
+* **Ephemeral Workflows**: Workflows run in an ephemeral mode, meaning that they are temporary and do not persist after the container restarts. This is suitable for development and testing.
+* **No Persistence**: In dev mode, there is no persistence for running workflow instances. All instance information is lost after a container restart, making it ideal for short-running or non-critical workflows.
+* **Development Focus**: Dev mode is designed for developers to gain experience and test workflows without the overhead of a full production environment.
+* **Hot-deployment of Workflows**: Developers can deploy new workflows by simply placing the workflow files in a designated folder, enabling rapid iteration and testing.
```

Comment:
Link to a guide that explains this?

Reply:
I'm not aware of such link. I'd be happy if someone with experience with dev-mode can create a blog to explain how to work with it.

Reply:
Agree that the dev mode isn't explained. 
Created a Jira story to track this [FLPATH-1451](https://issues.redhat.com/browse/FLPATH-1451)